Binary doesn't build in Windows11

### katana version:
main|dev

### Current Behavior:
Binary is not built correctly on Win11 (64bit) due to TreeSitter dependency

### Expected Behavior:
Building correctly

### Steps To Reproduce:
```
> go build .
github.com/smacker/go-tree-sitter/javascript: build constraints exclude all Go files in C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\javascript
# github.com/smacker/go-tree-sitter
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:17:18: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:21:21: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:25:20: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:30:26: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:34:20: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:38:32: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:43:9: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:46:18: undefined: Node
C:\Users\user\go\pkg\mod\github.com\smacker\go-tree-sitter@v0.0.0-20230720070738-0d0a9f78d8f8\iter.go:68:40: undefined: 
```

--- a/.goreleaser/windows.yml
+++ b/.goreleaser/windows.yml
@@ -15,7 +15,9 @@ builds:
     goos:
       - windows
     goarch:
-      - amd64
+      - 386
+      - arm64
+      - 386
 
 archives:
 - format: zip

--- a/.goreleaser/windows.yml
+++ b/.goreleaser/windows.yml
@@ -17,7 +17,7 @@ builds:
     goarch:
       - 386
       - arm64
-      - 386
+      - amd64
 
 archives:
 - format: zip

--- a/pkg/engine/parser/parser_generic.go
+++ b/pkg/engine/parser/parser_generic.go
@@ -1,4 +1,4 @@
-//go:build !386
+//go:build !(386 || windows)
 
 package parser
 

--- a/pkg/engine/parser/parser_nojs.go
+++ b/pkg/engine/parser/parser_nojs.go
@@ -1,4 +1,4 @@
-//go:build 386
+//go:build windows || 386
 
 package parser
 

--- a/pkg/utils/jsluice.go
+++ b/pkg/utils/jsluice.go
@@ -1,4 +1,4 @@
-//go:build !386
+//go:build !(386 || windows)
 
 package utils
 

--- a/pkg/utils/jsluice_test.go
+++ b/pkg/utils/jsluice_test.go
@@ -1,4 +1,4 @@
-//go:build !386
+//go:build !(386 || windows)
 
 package utils
 

